app/src/main/java/xdman/util/XDMUtils.java

```diff
@@ -458,8 +458,9 @@ public static String getDownloadsFolder() {
 				return path;
 			}
 		}
-		return new File(System.getProperty("user.home"), "Downloads")
-				.getAbsolutePath();
+		return new StringBuilder(System
+                .getProperty("user.home"))
+            .append(File.separator).append("Downloads").toString();

```

Comment:
Why? I see nothing wrong with the old method, and this method does not use absolute path, which can cause problems.

Reply:
Updated to use Paths, since it's more native than File and faster.